src/components/banner/GovBanner/GovBanner.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { ReactElement, useState } from 'react'
+import { Banner } from '../Banner/Banner'
+import { BannerHeader } from '../BannerHeader/BannerHeader'
+import { BannerFlag } from '../BannerFlag/BannerFlag'
+import { BannerButton } from '../BannerButton/BannerButton'
+import { BannerContent } from '../BannerContent/BannerContent'
+import { BannerGuidance } from '../BannerGuidance/BannerGuidance'
+import { BannerIcon } from '../BannerIcon/BannerIcon'
+import { MediaBlockBody } from '../../mediablock/MediaBlockBody/MediaBlockBody'
+import { BannerLockImage } from '../BannerLockImage/BannerLockImage'
+
+// assets
+import flagImg from 'uswds/src/img/us_flag_small.png'
+import dotGovIcon from 'uswds/src/img/icon-dot-gov.svg'
+import httpsIcon from 'uswds/src/img/icon-https.svg'
+
+type Language = 'english' | 'spanish'
+
+type TLD = '.gov' | '.mil'
+
+interface GovBannerCopy {
+  header: string
+  headerAction: string
+  tldSectionHeader: string
+  tldSectionContent: JSX.Element
+  httpsSectionHeader: string
+  httpsSectionContent: JSX.Element
+}
+
+const getCopy = (language: Language, tld: TLD): GovBannerCopy => {
+  const lock = <BannerLockImage title="Lock" description="A locked padlock" />
```

Comment:
thought: I'm wondering if we need to think about internationalization support differently as we add more languages. This isn't something I'm thinking for this PR or anything just starting to wonder as I review.

Reply:
Strongly agreed. Added a i18n label to capture what issues have occurred so far related to this topic https://github.com/trussworks/react-uswds/labels/i18n for reference.